### ISSUE TITLE
Import PIconnect fails when PI Asset Framework is not installed

### ISSUE THREAD

**Issue by Hugovdberg** [Hugovdberg/PIconnect#510]
Created from comment by @mpepera on #496:

I had a slightly different error. We are using an older PI Server without an Asset Framework implementation. Import PIconnect failed on the PIAF import database step. I just commented out the following line from `__init__.py` and happily went along my way!

`from PIconnect.PIAF import PIAFDatabase`

Cool project! Thanks for building it.

```
  File "C:\Users\myuser\AppData\Local\Continuum\anaconda3_32\lib\site-packages\PIconnect\__init__.py", line 26, in <module>
    from PIconnect.PIAF import PIAFDatabase

  File "C:\Users\myuser\AppData\Local\Continuum\anaconda3_32\lib\site-packages\PIconnect\PIAF.py", line 61, in <module>
    class PIAFDatabase(object):

  File "C:\Users\myuser\AppData\Local\Continuum\anaconda3_32\lib\site-packages\PIconnect\PIAF.py", line 67, in PIAFDatabase
    default_server = servers[AF.PISystems().DefaultPISystem.Name]

AttributeError: 'NoneType' object has no attribute 'Name'
```

_Originally posted by @mpepera in https://github.com/Hugovdberg/PIconnect/issues/496#issuecomment-591547348_

**Comment by Hugovdberg** [comment]
In a future release I will replace this with a conditional import, thanks for reporting this!

**Comment by Hugovdberg** [comment]
Upon further inspection it seems this should be solved by #501 as well.
